### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by ngocbd**
## Summary by CodeRabbit

- **Chores**
	- Updated `.gitignore` to ignore IDE files in the `.vscode/` directory.
- **Bug Fixes**
	- Disabled the functionality to read the owner's address from JSON configuration in the `BaseScript` contract.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "develop": {
     "11155111": {
-      "salt": 0
+      "salt": 0,
+      "owner": "0x921d360aD22A6D0289ce57fcb8250e299cB19EA3"
```
Comment (kevupton) [inline config.json]: no need for this change

**Review by kevupton — CHANGES_REQUESTED** [review]

File: config.json (hunk 1 above)
Comment (DuyAnhTran95) [reply config.json:5]: Forge project check pipeline failed without that line ATM

Comment (ngocbd) [reply config.json]: forge project check pipeline failed without that line

**Comment by kevupton** [comment]
I commented out the owner piece, as it was more example

**Review by kevupton — APPROVED** [review]

File: config.json (hunk 1 above)
Comment (kevupton) [reply config.json]: i just commented out this liine for now in the config function as it was purely just a helper method

**Pull request merged**